spec/lib/bolt_spec/conn.rb

```diff
@@ -2,18 +2,22 @@ module BoltSpec
   module Conn
     def conn_info(transport)
       tu = transport.upcase
-      default_port = case transport
-                     when 'ssh'
-                       20022
-                     when 'winrm'
-                       25985
-                     end
+      case transport
+      when 'ssh'
+        default_port = 2224
+        default_user = 'root'
+        default_password = 'root'
+      when 'winrm'
+        default_port = 55985
+        default_user = 'vagrant'
+        default_password = 'vagrant'
+      end
 
       {
         protocol: transport,
         host: ENV["BOLT_#{tu}_HOST"] || "localhost",
-        user: ENV["BOLT_#{tu}_USER"] || "vagrant",
-        password: ENV["BOLT_#{tu}_PASSWORD"] || "vagrant",
+        user: ENV["BOLT_#{tu}_USER"] || default_user,
+        password: ENV["BOLT_#{tu}_PASSWORD"] || default_password,
         port: ENV["BOLT_#{tu}_PORT"] || default_port,
         key: ENV["BOLT_#{tu}_KEY"] || Dir[".vagrant/**/private_key"]
```

Comment:
Does this need to change?

Reply:
I think I need to change the default ssh user to 'bolt', but since for now we plan to keep using vagrant for windows tests and docker for ssh tests I think this is the best solution? Alternatively we could add a vagrant user to the containers (shrug).